src/OrbitGl/App.cpp

```diff
@@ -1811,7 +1818,8 @@ orbit_base::Future<ErrorMessageOr<std::filesystem::path>> OrbitApp::RetrieveModu
 
   const auto it = modules_currently_loading_.find(module_id);
   if (it != modules_currently_loading_.end()) {
-    return it->second;
+    const ModuleLoadOperation& operation{it->second};
```

Comment:
Why the temporary?

Reply:
because `it->second.second` seemed unreadable. Since there is a struct now instead of a pair, its `it->second.future` which is better.